### ISSUE TITLE
Bring back dl.multinomial once we support unnormalized log probabilities.

### ISSUE THREAD

**Issue by nsthorat** [PAIR-code/deeplearnjs#721]
We don't currently support unnormalized log probabilities (that is, you have to call softmax on logits before passing to multinomial).

This is not compatible with TensorFlow, so let's remove it for 0.5 and then bring it back soon after.

**Comment by adarob** [comment]
Why can't this be left in since it works with softmax? This breaks MusicVAE, PerformanceRNN, etc.

**Comment by dsmilkov** [comment]
In deeplearn@next you can use either, so this bug is fixed.

You can pass logits:
```js
const logits = ...;
const sampledOutput = dl.multinomial(logits, numSamples);
```
or probabilities:

```js
const softmax = dl.softmax(logits);
const sampledOutput = dl.multinomial(softmax, numSamples, null /* seed */, true /* normalized */)
```
